charts/function-mesh-operator/templates/controller-manager-deployment.yaml

```diff
@@ -42,7 +42,11 @@ spec:
           - /manager
         args:
           - --enable-leader-election
-          - --config-file=/etc/config/configs.yaml
+          - --enable-pprof
```

Comment:
`--enable-pprof` havnt refer to the Values, we should add `--enable-pprof` only if `.Values.controllerManager.enable-pprof` is true.

Reply:
sorry, I was careless. Fixed.